tests/integration/abi_test.py

```diff
@@ -190,13 +192,17 @@ def test_roundtrip_abi_strategy(roundtrip_app):
 
     algod = get_algod()
     args = (rand,)
-    abi_arg_types = (abi_instance,)
-    abi_out_type = abi.TupleType([abi_instance] * 3)
 
     with open(filename) as f:
         roundtrip_teal = f.read()
 
-    inspector = DRE.dryrun_app(algod, roundtrip_teal, args, abi_arg_types, abi_out_type)
+    inspector = DRE.dryrun_app(
+        algod,
+        roundtrip_teal,
+        args,
+        abi_method_signature=method_sig,
+        omit_method_selector=True,
```

Comment:
I will not act on this suggestion but keep it open and link to it in #44. This is because sequencing the diff in this case does feel more complicated given all the upcoming changes.

Reply:
@tzaffi Works for me to address in #44 - resolving.

Reply:
Unresolving for easier spotting. Also, this is slated to be addressed in #45